### PR TITLE
[INLONG-4664][TubeMQ] Add GroupController and TopicBackendWorker

### PR DESCRIPTION
**Pull request by bluewang**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/incubator-inlong/issues) number)*

- Fixes https://github.com/apache/incubator-inlong/issues/4664

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/topic/TopicTdmController.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+@RestController
+@RequestMapping(path = "/business")
+@Slf4j
+public class TopicTdmController {
```
Comment (healchow) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/topic/TopicTdmController.java:43]: Excuse, what does the `Tdm` mean?

Comment (dockerzhang) [reply inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/topic/TopicTdmController.java:43]: @healchow do we still have the `business` concept?

Comment (bluewang) [reply inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/topic/TopicTdmController.java:43]: > Excuse, what does the `Tdm` mean?

tdm is the inlong manager

Comment (healchow) [reply inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/topic/TopicTdmController.java]: > @healchow do we still have the `business` concept?

We now use `inlong group` as data grouping and no longer support `business` concept.

Comment (healchow) [reply inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/topic/TopicTdmController.java]: > tdm is the inlong manager

@bluewang please change another name, tdm is so weird.

**Review by healchow — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**